e2e/pages/caseList.page.js

```diff
@@ -43,12 +43,13 @@ module.exports = {
     I.click(this.fields.search);
   },
 
-  searchForCasesWithName(caseName, state='Any') {
+  async searchForCasesWithName(caseName, state='Any') {
     this.setInitialSearchFields(state);
     // wait for our filters to load
     I.waitForVisible(this.fields.caseName, 30);
     I.fillField(this.fields.caseName, caseName);
     I.click(this.fields.search);
+    await I.runAccessibilityTest();
```

Comment:
not sure why accessibility reports needs to be done in arbitrary places, cannot be it done each time e.g page changed?